README.md

```diff
@@ -97,24 +97,9 @@ environments:
       - values/tags.yaml
 ```
 
-#### If you use Hashicorp Vault in AWS
+#### If you use Hashicorp Vault
 
-```helmyaml
-txSigner:
-  environment:
-    SECRET_STORE: "hashicorp"
-    VAULT_VERIFY: "true"
-    VAULT_MOUNT_POINT: "secrets"
-    VAULT_SECRET_PATH: "<KubernetesNameSpace>/keys"
-    VAULT_ADDR: https://vault
-    VAULT_CACERT: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
-    VAULT_SKIP_VERIFY: true
-```
-
-- `IAMRole`: Amazon Resource Names (ARN) of AWS IAM role (string).
-- `Region`: AWS Region in which resources are created (string).
-- `KMSKeyId`: AWS KMS key ID to use for encryption and decryption (string).
-- `SecretId`: AWS Secret manager's alias or name where the root token is stored (string).
+Apply the `vaults/REAMDE.md` to deploy Harshicorp Vault and configure Orchestrate
```

Comment:
MINOR: Provide a link to the actual file [vaults/README.md](/vaults/README.md)

Reply:
good point